index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Happy International Women’s Day</title>
+    <link rel="stylesheet" href="css/main.css">
```

Comment:
Can you confirm that the link to your CSS is correct?

Reply:
The main.css inside the css directory is the linked styling file.

"css/main.css" works on my VScode, switching to "./css/main.css" works as well.
<img width="343" alt="image" src="https://user-images.githubusercontent.com/35523688/225388303-19c92d19-91c7-4fd9-97c6-dbd11b7635ce.png">